src/components/Editor/Tiptap.tsx

```diff
@@ -0,0 +1,137 @@
+"use client";
+import { BubbleMenu, EditorContent, FloatingMenu, useEditor } from "@tiptap/react";
+import parse from "html-react-parser";
+import { Heading1, Heading2, Heading3, List, ListOrdered } from "lucide-react";
+import StarterKit from "@tiptap/starter-kit";
+import { BoldIcon, Code, CodeSquare, Italic, StrikethroughIcon } from "lucide-react";
+import React, { useEffect, useState } from "react";
+import EditorMenubar from "./EditorMenuBar";
+import { Codepen } from "react-feather";
+
+interface EditorProps {
+  editorState: string;
+  setEditorState: (params: string) => void;
+}
+export default ({ editorState, setEditorState }: EditorProps) => {
```

Comment:
Please add the function name as it will act as a component and this might give problem at production

Reply:
I'll do it ..